backend/config/secrets.yml

```diff
@@ -8,7 +8,7 @@ development:
   qualtrics_api_url: https://iad1.qualtrics.com/API/v3
   qualtrics_group_library: 'GR_7R5NeLAPtj22K46'
   qualtrics_launch_url: <%= ENV.fetch('QUALTRICS_LAUNCH_URL', 'https://riceuniversity.co1.qualtrics.com/jfe/form') %>
-  qualtrics_template_survey_id: '<%= ENV.fetch('QUALTRICS_TEMPLATE_SURVEY_ID', 'SV_9Kz6X9nM1q31mzI') %>'
+  qualtrics_template_survey_id: '<%= ENV.fetch('QUALTRICS_TEMPLATE_SURVEY_ID', 'SV_cA5YICrqMwDcNXU') %>'
```

Comment:
cloned the template survey on qualtrics and set up a [simple empty dev survey](https://riceuniversity.co1.qualtrics.com/survey-builder/SV_cA5YICrqMwDcNXU/edit) for this purpose